README.md

```diff
@@ -55,6 +55,13 @@ tox -e behave features/unattached_commands.feature
 tox -e behave features/unattached_commands.feature:55
 ```
 
+When developing/debugging a new scenario:
+
+ 1. Add a `@wip` tag decorator on the scenario
+ 2. Run `tox -e behave -- -w`   # will only run @wip scenarios with verbose logs
```

Comment:
You don't need the # comment sign this isn't code :)

2. To only run @wip scenarios then run: `tox -e behave -- -w`